docs/agents/ucsc_radiometer.rst

```diff
@@ -0,0 +1,70 @@
+.. highlight:: rst
+
+.. _ucsc_radiometer:
+
+==============
+UCSC Radiometer Agent
+==============
+
+The UCSC Radiometer agent uses HTTP queries to publish pwv data from a Flask server.
```

Comment:
The phrasing of this makes it sound like the agent is using HTTP to publish the pwv data, which isn't strictly true. Perhaps rephrase to "The UCSC Radiometer Agent monitors the PWV through the UCSC Radiometer web server."